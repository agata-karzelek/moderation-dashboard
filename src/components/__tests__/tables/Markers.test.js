import React from "react";
import { shallow } from "enzyme";
import Markers from "../../tables/Markers";

describe("Test Cases for Markers", () => {
  test("Snapshot", () => {
    const wrapper = shallow(<Markers />);
    expect(wrapper).toMatchSnapshot();
  });
});
