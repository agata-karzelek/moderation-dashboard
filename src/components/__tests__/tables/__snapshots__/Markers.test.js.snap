// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Cases for Markers Snapshot 1`] = `<Fragment />`;
